src/js/button.js

```diff
@@ -12,6 +12,13 @@ import {assign} from './utils/obj';
  * @extends ClickableComponent
  */
 class Button extends ClickableComponent {
+  constructor(player, options) {
+    super(player, options);
+
+    this._mouseused = false;
```

Comment:
You're right. That would be better since I want this as self contained as possible.